Mvc.sln

```diff
@@ -125,6 +125,8 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "SimpleWebSite", "test\WebSi
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "SecurityWebSite", "test\WebSites\SecurityWebSite\SecurityWebSite.xproj", "{D28CAC79-7004-4B69-993B-EDEB4653BFA8}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "EnumDisplay", "test\WebSites\EnumDisplay\EnumDisplay.xproj", "{E65CCD64-22CB-4883-A26A-367FDC1DD4C7}"
+EndProject
```

Comment:
We have so many test projects. Please add the new `enum`, action, and views to an existing site e.g. HtmlGenerationWebSite.